lib/python/pyflyby/_importstmt.py

```diff
@@ -391,20 +390,20 @@ def _from_ast_node(cls, node):
           `ImportStatement`
         """
         if isinstance(node, ast.ImportFrom):
-            if isinstance(node.module, str):
-                module = node.module
-            elif node.module is None:
-                # In python2.7, ast.parse("from . import blah") yields
+            if node.module is None:
+                # In python2.7 and 3 as well, ast.parse("from . import blah") yields
```

Comment:
I reckon this is not just the case for Python 2.7:

In Python3:

```py
In [1]: from pyflyby._parse import PythonStatement

In [2]: s = PythonStatement("from . import xrefs")

In [3]: s.ast_node.module

In [4]: type(s.ast_node.module)
Out[4]: NoneType
```

I have added a commit to address this, let me know what do you think?

Reply:
Thanks for digging into this.